core/dbt/cli/params.py

```diff
@@ -761,3 +761,10 @@ def _version_callback(ctx, _param, value):
     help="Whether or not to write the manifest.json and run_results.json files to the target directory",
     default=True,
 )
+
+upload_artifacts = click.option(
+    "--upload-artifacts/--no-upload-artifacts",
+    envvar="DBT_UPLOAD_ARTIFACTS",
```

Comment:
Are we using a new pattern for naming our env vars now?  @QMalcolm can you speak to this?

Reply:
We talked offline, and the env var pattern for `_DBT_` is not in use.
We decided to go with  more specific env var to avoid collision